cogs5e/initiative/cog.py

```diff
@@ -967,7 +967,11 @@ async def effect(self, ctx, target_name: str, effect_name: str, *args):
         targets = []
 
         for i, t in enumerate([target_name] + args.get("t")):
-            target = await combat.select_combatant(ctx, t, f"Select target #{i + 1}.", select_group=True)
+            target = await combat.select_combatant(
+                t,
+                f"Pick your {i+1}{['th','st','nd','rd','th'][(n // 10 % 10 != 1) * min(n % 10, 4)]} target.",
```

Comment:
readability counts! make this an `ordinal()` function without wacky indexing

Reply:
Do you think it should go in utils then?

Reply:
created the function and replaced the instances where this mess was used.